### DISCUSSION TITLE
v0.9.0

### DISCUSSION THREAD

**Discussion by 0xCUB3** [2009-Nissan-Cube/About-This-Hack#37] [Announcements]
The final major release before stable...maybe...👀

## What's Changed
- 🎉CUT LAUNCH TIME IN HALF 🎉
- Added Dual-GPU Support
- Cleaned Up the Code by @nocontent06 in https://github.com/0xCUB3/About-This-Hack/pull/36
- Temporarily Removed Localizations to Improve Performance
- Changed How Commands are Handled
- Added Model Identifier by @nocontent06 in https://github.com/0xCUB3/About-This-Hack/pull/35

*Please note that About This Hack is still in beta, so bugs exist. If you do find anything that isn't covered in the README, please [open an issue](https://github.com/0xCUB3/About-This-Hack/issues).*

**Full Changelog**: https://github.com/0xCUB3/About-This-Hack/compare/0.8.1...0.9.0

<hr /><em>This discussion was created from the release <a href='https://github.com/0xCUB3/About-This-Hack/releases/tag/0.9.0'>v0.9.0</a>.</em>

**Comment by 0xCUB3** [comment]
What are your thoughts on this release?

**Comment by MDNich** [comment]
Impressed with improvements, glad to see the project is nearing stable !

**Reply by 0xCUB3** [reply]
thanks a lot!

**Comment by kaoskinkae64** [comment]
the indication of the essential compiled version for those of us who use the betas has disappeared

**Reply by 0xCUB3** [reply]
> the indication of the essential compiled version for those of us who use the betas has disappeared

What do you mean by this?
